beluga_amcl/config/Amcl.cfg

```diff
@@ -91,14 +91,14 @@ gen.add(
     "recovery_alpha_slow", double_t, 0,
     "Exponential decay rate for the slow average weight filter, "
     "used in deciding when to recover by adding random poses.",
-    default=0.001, min=0., max=1.
+    default=0, min=0., max=1.
```

Comment:
@nahueespinosa meta: I don't think this quite matches AMCL defaults in ROS 1, see https://github.com/ros-planning/navigation/blob/9ad644198e132d0e950579a3bc72c29da46e60b0/amcl/src/amcl_node.cpp#L447-L448. These are set to zero in https://github.com/ros-planning/navigation/blob/9ad644198e132d0e950579a3bc72c29da46e60b0/amcl/cfg/AMCL.cfg#L23-L24, but ignored on first reconfigure call (https://github.com/ros-planning/navigation/blob/9ad644198e132d0e950579a3bc72c29da46e60b0/amcl/src/amcl_node.cpp#L521-L528).

Old AMCL code is... peculiar. Careful.

Reply:
Fixed in https://github.com/Ekumen-OS/beluga/pull/244/commits/822ddbaae0c3f7da9f9d7eabc1a3569ba7c4838f.